### ISSUE TITLE
I had error with PyInstaller on exe file

### ISSUE THREAD

**Issue by uchidaka** [pyinstaller/pyinstaller#4917]
Please help me.
I'm a begginer for Python.
I want to make image processing application as .exe files for distribution.
I made 11 lines small test program (err_imshow.py/err_imshow.exe)
This program is working perfectly on IDE.(Anacond3>Spyder)

But when i made exe file by pyinstaller on console,
(typed : pyinstaller err_imshow.py)
the compile was success, err_imshow.exe was genelated.

but 
Line9:cv2.imshow("saved_Canny",img2) 
caused error.

The error messages is 
```
[qt.qpa.plugin: Could not find the Qt platform plugin "windows" in ""
This application failed to start because no Qt platform plugin could be initialized. 
Reinstalling the application may fix this problem.]
```
Canny.jpg was genelated.And img2 was not displayed.
So,Line9:cv2.imshow("saved_Canny",img2) 
didn't work.

The environments are
 Windows10 64bit/Python 3.7.7/openCV 4.2.0/pyinstaller 3.6

-----------------------------------------------
```
err_imshow.py (err_imshow.exe)

import cv2
from PIL import ImageGrab
ImageGrab.grab().save("my_pc_desktop.jpg")
img = cv2.imread("my_pc_desktop.jpg")
print(img.shape)
gry = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
img2 = cv2.Canny(gry, 30, 200)
cv2.imwrite("Canny.jpg", img2)
cv2.imshow("saved_Canny",img2)
cv2.waitKey()
cv2.destroyAllWindows()
```

**Comment by Legorooj** [comment]
Are you running the executable under the `dist` folder?

**Comment by uchidaka** [comment]
Yes,I'm running on dist folder it.
dist\err_imshow\err_imshow.exe
So It's working until Line 8. (cv2.imwrite("Canny.jpg",img2)
And i can see 37 pieces opencv*.dll in same folder.
following...
opencv_aruco420.dll
opencv_bgsegm420.dll
opencv_calib3d420.dll
opencv_ccalib420.dll
opencv_core420.dll
opencv_dnn420.dll
opencv_face420.dll
opencv_features2d420.dll
opencv_flann420.dll
opencv_fuzzy420.dll
opencv_hfs420.dll
opencv_highgui420.dll
opencv_img_hash420.dll
opencv_imgcodecs420.dll
opencv_line_descriptor420.dll
opencv_ml420.dll
opencv_objdetect420.dll
opencv_optflow420.dll
opencv_phase_unwrapping420.dll
opencv_photo420.dll
opencv_plot420.dll
opencv_quality420.dll
opencv_reg420.dll
opencv_rgbd420.dll
opencv_saliency420.dll
opencv_shape420.dll
opencv_stitching420.dll
opencv_structured_light420.dll
opencv_surface_matching420.dll
opencv_text420.dll
opencv_tracking420.dll
opencv_video420.dll
opencv_videoio420.dll
opencv_xfeatures2d420.dll
opencv_ximgproc420.dll
opencv_xphoto420.dll

I appreciate your reply.

**Comment by Legorooj** [comment]
What's the ouput of `pip freeze` in the environment?

**Comment by uchidaka** [comment]
I am using Ananonda 3 2020.02.
I've read an article saying Anaconda has a lot of problems using pip and I'm not using pip. Therefore, when I introduced opencv4.2, I wrote conda install -c conda-forge opencv=4.2.0 on console instead of pip.
(Pip 20.0.2 may be available.)
For the warning at compile time, lib not found:mpich2mpi.dll,
There was no error except that pgf90.dll, pgc14.dll, impi.dll, msmpi.dll were displayed.
At that time the compilation was successful and the exe worked fine until line8.
We added pyautogui0.94.8 and pillow7.0.0 to environments for studying yesterday and today.
Furthermore, I installed Microsoft.WebDriver and added selenium to the environment by conda install -c conda-forge selenium.
And when I compiled err_imshow.py to get the compile-time warning message in response to your reply, today I got a compilation error and the exe was not generated.
The error message is _int_.py,version.py,parser.py,feedparser.py,_policybace.py,
In utils.py,random.py, TypeError:'module' object is not callable pre-safe-import-module hook failed,needs fixing.
It was that.
However, err_imshow.py is working perfectly on the IDE (Spyder 4.1.3).
